### PR TITLE
feat: pr template and contributing

### LINKED ISSUE #864: Support GitHub recommended community standards
**Context**
When checking in the Insights tab of the GibHub, we can notice we are missing a few things to be well supporting recommended community standards.

**To do**

- [ ] add Code of conduct
- [ ] add Contributing guide
- [ ] add a Pull request template

### PR DESCRIPTION
**Pull request by quentinderoubaix**
closes #864

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Before creating the pull request, please make sure you do the following:
+
+- Read the Contributing Guidelines at https://github.com/AmadeusITGroup/AgnosUI/blob/main/CONTRIBUTING.md.
+- Check that there isn't already a PR that solves the problem the same way. If you find a duplicate, please help us reviewing it.
+- Update the corresponding documentation if needed.
+- Include relevant tests that fail without this PR but pass with it.
+
```
Comment (andreascorti) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: Shouldn't we include some basic tasks in the template?
For example title well formatted, issue linked, context, explanation of the solution, which packages include

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+We are always looking for quality contributions and will be happy to accept your Pull Requests.
+
+AgnosUI is a powerful library that provides configurable Typescript component factories, headless wrappers and ready-to-use components compliant to the [CSS Bootstrap design](https://getbootstrap.com/).
+
+The [Open Source Guides](https://opensource.guide/) website has a collection of resources for individuals, communities, and companies. These resources help people who want to learn how to run and contribute to open source projects. Contributors and people new to open source alike will find the following guides especially useful:
+
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+- [Building Welcoming Communities](https://opensource.guide/building-community/)
+
```
Comment (andreascorti) [inline CONTRIBUTING.md:11]: Shouldn't include the minimum expectation for a contribution?
For example, if someone wants to contribute a new component (very corner use case but...), should it include core + all frameworks? headless + bootstrap? same for bugs /issues, do we accept a PR for only a specific framework?

**Review by andreascorti — COMMENTED** [review]
I think we should be a bit more explicit about what we expect, especially for cross-framework issues

**Review by andreascorti — APPROVED** [review]
LGTM thanks @quentinderoubaix

**Comment by quentinderoubaix** [comment]
thx for the review @andreascorti !

**Pull request merged**